src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -3,11 +3,29 @@
 {# Widgets #}
 
 {% block money_widget -%}
-    {% if not valid %}
-        {% set group_class = ' form-control is-invalid' %}
-        {% set valid = true %}
-    {% endif %}
-    {{- parent() -}}
+    {%- if not valid -%}
+        {%- set group_class = ' form-control is-invalid' -%}
+        {%- set valid = true -%}
+    {%- endif -%}
```

Comment:
This if statement should be removed. We do not want to add `{%- set group_class = ' form-control is-invalid' -%}` nor do we want `{%- set valid = true -%}`. 

With this if-statement (current code): 
<img width="877" alt="screen shot 2018-02-10 at 12 36 43" src="https://user-images.githubusercontent.com/1275206/36061754-11425622-0e5f-11e8-9d8f-c9708104b959.png">

When the if-statement is removed: 
<img width="874" alt="screen shot 2018-02-10 at 12 35 35" src="https://user-images.githubusercontent.com/1275206/36061758-1f124f64-0e5f-11e8-93b3-0877d980ef1e.png">